lib/resque_bus/subscriber.rb

```diff
@@ -0,0 +1,28 @@
+module QueueBus
```

Comment:
We should `require 'queue_bus/subscriber'` to guarantee it's loaded. Otherwise if this loads first, `instance_method(:perform)` won't exist yet.